shared/src/common/enums/api/bs-api-path.enum.ts

```diff
@@ -1,6 +1,8 @@
 enum BSApiPath {
   ROOT = '/',
   SPACES = '/spaces',
+  SPACE = '/spaces/',
```

Comment:
```suggestion
```
should be without space at the end

Reply:
but I created this enum to have slash in the end, without slash it won't work
![image](https://user-images.githubusercontent.com/86227025/155296248-788c7794-37db-400c-9012-f441a2dbffd4.png)
So, how should I add a slash in route?

Reply:
add a new api enum for spaces